iree_tests/configs/models_cpu_llvm_task.json

```diff
@@ -2,20 +2,19 @@
     "config_name": "cpu_llvm_task",
     "iree_compile_flags" : [
       "--iree-hal-target-backends=llvm-cpu",
-      "--iree-llvmcpu-target-cpu-features=host"
+      "--iree-llvmcpu-target-cpu-features=host",
+      "--iree-input-demote-i64-to-i32"
     ],
     "iree_run_module_flags": [
       "--device=local-task"
     ],
-    "skip_compile_tests": [
-      "pytorch/models/sdxl-scheduled-unet-3-tank",
-      "pytorch/models/sdxl-vae-decode-tank",
-      "pytorch/models/sdxl-prompt-encoder-tank"
-    ],
+    "skip_compile_tests": [],
     "skip_run_tests": [],
     "expected_compile_failures": [
-      "pytorch/models/opt-125M", // TODO(#17344): need to regenerate .mlirbc
-      "pytorch/models/resnet50", // TODO(#17344): need to regenerate .mlirbc
+      "pytorch/models/direct/opt-125M", // TODO(#17344): need to regenerate .mlirbc
+      "pytorch/models/direct/resnet50", // TODO(#17344): need to regenerate .mlirbc
     ],
-    "expected_run_failures": []
+    "expected_run_failures": [
+      "pytorch/models/onnx-export/mobilebert-uncased",
+    ]
```

Comment:
> This commit adds testing for 4 models that we have noticed to be unstable and want to keep passing through CI

Can we add models that we think are _stable_ too? :) I'd like to prefetch all models from e2eshark and get them running here if possible, rather than cherrypick a few _after_ we notice issues.